core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -9,13 +9,12 @@ class DefaultPageBuilder(
 ) : PageBuilder {
 
     override fun pageForModule(m: Module): ModulePageNode =
-        ModulePageNode("root", contentForModule(m), m, m.packages.map { pageForPackage(it) })
+        ModulePageNode(m.name ?: "root", contentForModule(m), m, m.packages.map { pageForPackage(it) })
```

Comment:
It won't be null anymore

Reply:
But you might actually want to check for emptiness, and then put "root" as a name

Reply:
Done.